src/vendor.browser.ts

```diff
@@ -12,9 +14,36 @@ import '@angular/forms';
 import '@angular/http';
 import '@angular/router';
 
+// Angular2 Material
+import '@angular2-material/core';
```

Comment:
@binarious i realize now that the material components are also being included in the main bundle of the production build as well, don't know how i missed in the first place...thanks for the tip, i am updating the pr now